apps/api/v2/src/modules/oauth-clients/controllers/oauth-clients/oauth-clients.controller.ts

```diff
@@ -98,6 +107,21 @@ export class OAuthClientsController {
     return { status: SUCCESS_STATUS, data: client };
   }
 
+  @Get("/managed-users/:clientId")
+  @HttpCode(HttpStatus.OK)
+  @Roles([MembershipRole.ADMIN, MembershipRole.OWNER, MembershipRole.MEMBER])
```

Comment:
If MEMBER is required, do we need the others? cc @supalarry

Reply:
looking at the code of apps/api/v2/src/modules/auth/guards/organization-roles/organization-roles.guard.ts it is required yes

Reply:
As Morgan noted above - we indeed need to explicitly list all roles.